js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -61,15 +61,17 @@ export const useRunsForTimeline = ({
   const {localCacheIdPrefix} = useContext(AppContext);
   const completedRunsCache = useMemo(() => {
     if (filter) {
-      return new HourlyDataCache<RunTimelineFragment>(
-        localCacheIdPrefix ? `${localCacheIdPrefix}-useRunsForTimeline-filtered` : false,
-        JSON.stringify(filter),
-        3,
-      );
+      return new HourlyDataCache<RunTimelineFragment>({
+        id: localCacheIdPrefix ? `${localCacheIdPrefix}-useRunsForTimeline-filtered` : false,
+        keyPrefix: JSON.stringify(filter),
+        keyMaxCount: 3,
+        version: 1,
```

Comment:
Track this value in a single module-level const (with a loud comment?) so that we don't miss changing it in both places?